pallets/rmrk-core/src/lib.rs

```diff
@@ -335,8 +348,7 @@ pub mod pallet {
 			collection_id: T::CollectionId,
 		},
 		PropertySet {
-			collection_id: T::CollectionId,
-			maybe_nft_id: Option<T::ItemId>,
+			entity: EntityOf<T>,
```

Comment:
The changes will cause chain indexing problems for PhalaWorld. @h4x3rotab do you know a best practice to enable this `Entity` feature so we can ensure that this change doesn't break our indexers for PW and Stakepool?